sdk/servicebus/service-bus/test/utils/envVarUtils.ts

```diff
@@ -76,7 +79,7 @@ function getEnvVarValue(name: string, forBrowser?: boolean): string | undefined
       defaultValue = "unpartitioned-queue-sessions";
       break;
     case EnvVarKeys.TOPIC_NAME:
-      defaultValue = "partitioned-topic";
+      defaultValue = "new-partitioned-topic";
```

Comment:
Why the change for default value for partitioned topics?

Reply:
There's a thread with Eng Sys on Teams about this, something seems to be wrong with the Service Bus namespace on service side. The tests weren't passing when working with the previously configured name- can take a look at the previous builds.
This one's a successful build since using a different name - https://dev.azure.com/azure-sdk/internal/_build/results?buildId=216624

Reply:
We could revert this only if the namespace is fixed, else we'll need this change in for test / release. Thoughts? Is this blocking and needs an issue tracking it instead? Or do we want to use a better new name?

Reply:
Why do the tests fail? Can we address that instead of updating this?

Reply:
Please see the related Teams thread that was opened on it. It's a service side issue and not related to SDK.

Reply:
Since we don't have a resolution to the bug yet, understanding is that it's likely going to take longer.
Hence updated PR with workaround to unblock for now. We likely need to engage service team to see why that particular subscription entity seems to have gone corrupt. Filed https://github.com/Azure/azure-sdk-for-js/issues/6682

Reply:
A new name is fine as a workaround. If you plan to rename it back to the original name, please file an issue for that.

Reply:
Yes, planning to use https://github.com/Azure/azure-sdk-for-js/issues/6682 to track any reactions to fix that may come out of service side.